src/App.jsx

```diff
@@ -137,13 +139,35 @@ function KanbanBoard() {
   const handleTaskUpdate = async (taskId, updates) => {
     if (!selectedTask?.columnId) return;
     await updateTask(selectedTask.columnId, taskId, updates);
+    setSelectedTask(prev => ({
+      ...prev,
+      ...updates
+    }));
   };
 
   const handleTaskDelete = async (taskId) => {
     if (!selectedTask?.columnId) return;
     await deleteTask(selectedTask.columnId, taskId);
   };
 
+  const handleAddTask = async (columnId) => {
+    if (!newTaskContent?.trim()) return;
+    
+    const dueDate = new Date();
+    dueDate.setDate(dueDate.getDate() + 7); // Default to 1 week from now
+    
+    await addTask(columnId, {
+      id: crypto.randomUUID(),
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
id: crypto.randomUUID?.() || String(Date.now() + Math.random()),
```
Add error handling for the crypto.randomUUID() call as it might not be available in all browsers, potentially causing task creation to fail silently.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;